crates/biome_js_analyze/src/aria_analyzers/a11y/no_aria_unsupported_elements.rs

```diff
@@ -120,4 +122,35 @@ impl Rule for NoAriaUnsupportedElements {
             }),
         )
     }
+
+    fn action(_ctx: &RuleContext<Self>, _state: &Self::State) -> Option<JsRuleAction> {
+        let element = _ctx.query();
+        let mut mutation = _ctx.root().begin();
+        let mut removed_attribute = "role".to_string();
```

Comment:
Do we really need to allocate the string?
```suggestion
        let mut removed_attribute = "role";
```